otx/cli/tools/train.py

```diff
@@ -141,6 +141,11 @@ def get_args():
         default=None,
         help="The data.yaml path want to use in train task.",
     )
+    parser.add_argument(
+        "--auto-decrease-bs",
+        action="store_true",
+        help="Decrease batch size if current batch size isn't fit to CUDA memory.",
+    )
```

Comment:
Same here. If we enable this via `ConfigurableParameters` we don't need new CLI param.